src/tests/integration/test-data/foundry/test/SetUpTest.t.sol

```diff
@@ -38,4 +42,8 @@ contract SetUpTest is Test {
         // x is used and not thrown away by the optimizer
         assertEq(x, x);
     }
+
+    function testSetupData() public {
```

Comment:
This test fails on `master` and passes on this branch. It effectively states that the constraint on a contract storage variable established in `setUp` is propagated to the main test.